packages/docs/src/public/vuetify-docs-sw.js

```diff
@@ -0,0 +1,52 @@
+const CACHE = 'docs'
+const assets = ['/']
+
+self.addEventListener('install', event => {
+  event.waitUntil(
+    caches.open(CACHE).then(cache =>
+      cache.addAll(assets))
+  )
+})
+
+self.addEventListener('fetch', function (event) {
+  const response = fromCache(event.request) // try cache
+    .then(response => response)
+    .catch(() => fromServer(event.request) // try server
+      .then(response => response).catch(() => fallback()) // generic fallback
+    )
+    // Only fetch if asset is from origin
+  if (event.request.url.startsWith(self.location.origin)) {

```

Comment:
Yeah, I believe i put that in there because most of the 3rd party requests would throw fetch error due to cross-origin policies, or get rejection of the request by the 3rd party.